src/execSync.js

```diff
@@ -1,9 +1,18 @@
 const childProcess = require('child_process');
 
-module.exports = (command) => {
-  const options = {};
+/**
+ * Runs a command synchronously forwarding the stdout and stderr to the current process'
+ * stdout and stderr.
+ *
+ * @param {string} command - The command to be executed
+ * @param {*} options - An object with options to be passed to childProcess.execSync
+ * @param {*} env - An object containing a set of env. variables to extend the process'.
+ */
+module.exports = (command, options = {}, env = {}) => {
+  const extendedOptions = { ...options };
   if (!process.env.JEST_WORKER_ID) {
-    options.stdio = 'inherit';
+    extendedOptions.stdio = 'inherit';
   }
-  childProcess.execSync(command, options);
+  extendedOptions.env = { ...env, ...process.env };
+  childProcess.execSync(command, extendedOptions);
```

Comment:
@gonzaloriestra APIs for triggering processes usually support modifying the working directory and the environment variables used. [Here's](https://nodejs.org/api/child_process.html#child_processexeccommand-options-callback) an example of the options you can pass to `exec` and `execSync`. Although doing `cd` and prefixing the command with `FOO=BAR`, it's more reliable to use their APIs and let the internals map that to the shell of the operative system in which the CLI runs.